src/common/common-core/src/main/java/ee/ria/xroad/common/util/TimeUtils.java

```diff
@@ -39,6 +42,8 @@
  * This class contains various time related utility methods.
  */
 public final class TimeUtils {
+    private static Clock clock = Clock.systemUTC();
```

Comment:
Is utc correct here? I think by default it was using Clock.systemDefaultZone()

Reply:
fixed